packages/core/src/components/dropdown/index.js

```diff
@@ -0,0 +1,353 @@
+import {
```

Comment:
Please write tests for this component. See other tests for examples.

This dropped our coverage by ~20%!

Reply:
done